pkg/restapi/operation/operations.go

```diff
@@ -137,48 +139,73 @@ func New(config *Config) (*Operation, error) {
 		oidcClientSecret: config.OIDCClientSecret,
 		oidcCallbackURL:  config.OIDCCallbackURL,
 	}
+	// TODO remove this empty string check. This is just here temporarily to allow for testing of other parts
```

Comment:
FYI @talwinder50 @llorllale

Reply:
I did this so that bootstrap stuff (store creation, handlers) can be tested while @talwinder50 works on the OIDC stuff.